compiler/ballerina-parser/src/main/java/io/ballerina/compiler/syntax/tree/SyntaxKind.java

```diff
@@ -133,6 +123,16 @@ public enum SyntaxKind {
     RE_KEYWORD(400, "re"), // Any kind above this is considered as a keyword
     GROUP_KEYWORD(401, "group"),
     COLLECT_KEYWORD(402, "collect"),
+    WHERE_KEYWORD(403, "where"),
+    JOIN_KEYWORD(404, "join"),
+    OUTER_KEYWORD(405, "outer"),
+    EQUALS_KEYWORD(406, "equals"),
+    ORDER_KEYWORD(407, "order"),
+    BY_KEYWORD(408, "by"),
+    ASCENDING_KEYWORD(409, "ascending"),
+    DESCENDING_KEYWORD(410, "descending"),
```

Comment:
In the error handler seems we don't have these two entries under the `isKeyword()` method.